controller/array_action/config.py

```diff
@@ -1,10 +1,12 @@
 ISCSI_CONNECTIVITY_TYPE = "iscsi"
 FC_CONNECTIVITY_TYPE = "fc"
-SPACE_EFFICIENCY_THIN = 'thin'
-SPACE_EFFICIENCY_COMPRESSED = 'compressed'
-SPACE_EFFICIENCY_DEDUPLICATED = 'deduplicated'
-SPACE_EFFICIENCY_THICK = 'thick'
-SPACE_EFFICIENCY_NONE = 'none'
+SPACE_EFFICIENCY_THIN = "thin"
+SPACE_EFFICIENCY_COMPRESSED = "compressed"
+SPACE_EFFICIENCY_DEDUPLICATED = "deduplicated"
+SPACE_EFFICIENCY_DEDUPLICATED_COMPRESSED = "dedup_compressed"
+SPACE_EFFICIENCY_DEDUPLICATED_THIN = "dedup_thin"
```

Comment:
```suggestion
SPACE_EFFICIENCY_THIN = "thin"
SPACE_EFFICIENCY_COMPRESSED = "compressed"
SPACE_EFFICIENCY_DEDUPLICATED = "deduplicated"
SPACE_EFFICIENCY_DEDUPLICATED_THIN = "dedup_thin"
SPACE_EFFICIENCY_DEDUPLICATED_COMPRESSED = "dedup_compressed"
```
to keep the thin-compressed order consistent

Reply:
same [here](https://github.com/IBM/ibm-block-csi-driver/blob/d3de8b692bdca61e8f014aac186ba47bdd29c2fb/controller/array_action/array_mediator_svc.py#L361-L362)

Reply:
done